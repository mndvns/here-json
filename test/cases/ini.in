./bin/here-json <<EOF
[what]
in = 10
the = 11
world = is this format
EOF