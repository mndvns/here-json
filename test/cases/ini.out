{"what":{"in":"10","the":"11","world":"is this format"}}